### ISSUE TITLE
1. Security (Non Functional)

### ISSUE THREAD

**Issue by Farhana0088** [algonquin-college-sat/24s-cst8319-320-team-4#35]

**Comment by Farhana0088** [comment]
<img width="1443" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1Y1QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--fe9484a3be0ffeec9b0ea6addad1ca4bc6d1132f/secured%20website.jpg" alt="secured website.jpg" />

**Comment by Farhana0088** [comment]
The website is now secured. 

We guided the client to purchase an SSL certificate. and we helped them activate the SSL certificate. We worked on the Namecheap C panel to activate the SSL certificate and fixed the numbers and code.<img width="1471" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK3A1QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--463d98f585b194afca2db3019ae392d14ea1541b/cpannel_fix.jpg" alt="cpannel_fix.jpg" />
